rasa/cli/scaffold.py

```diff
@@ -216,6 +216,9 @@ def run(args: argparse.Namespace) -> None:
         # Can't use `if not path` either, as `None` will be handled differently (abort)
         if path == "":
             path = "."
+            
+        if "win" not in sys.platform.lower():
```

Comment:
Hi! Thanks for creating this!! 🚀

I noticed that this won't work for macos `darwin` because it also contains `"win"` 🙈
Is there another way to check this? I think `platform.system()` will output `windows` for windows?

Reply:
also can you create a changelog entry as described [here](https://github.com/RasaHQ/rasa/tree/main/changelog)? that would be awesome!!

Reply:
Can you also create a test similar to [this one](https://github.com/RasaHQ/rasa/blob/88d97e1d54ce79b9dac2d1091c7db7eb80fc6451/tests/cli/test_rasa_init.py#L30-L33) in the same file `tests/cli/test_rasa_init.py`?

Reply:
Thanks for your review! I'll get your feedback incorporated in the next commit.